ambuda/views/auth.py

```diff
@@ -79,51 +91,99 @@ def _is_valid_reset_token(row: db.PasswordResetToken, raw_token: str, now=None):
     return True
 
 
+# Copied from https://wtforms.readthedocs.io/en/2.3.x/validators/
+class FieldLength(object):
+    def __init__(self, min=-1, max=-1, message=None):
+        self.min = min
+        self.max = max
+        if not message:
+            message = "Field must be between %i and %i characters long." % (min, max)
+        self.message = message
+
+    def __call__(self, form, field):
+        field_len = field.data and len(field.data) or 0
+        if field_len < self.min or self.max != -1 and field_len > self.max:
```

Comment:
`self.max and field_len > self.max` if using `None` per above

Reply:
If sentinel values are initialized as integers at the top, the logic that follows can safely run integer operations.